apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -0,0 +1,284 @@
+import React, {Component, PropTypes} from 'react';
+import color from "../../util/color";
+import {Table, sort} from 'reactabular';
+import wrappedSortable from '../tables/wrapped_sortable';
+import orderBy from 'lodash/orderBy';
+import PasswordReset from './PasswordReset';
+import ShowSecret from './ShowSecret';
+import i18n from "@cdo/locale";
+
+export const studentSectionDataPropType = PropTypes.shape({
+  id: PropTypes.number,
+  name: PropTypes.string,
+  username: PropTypes.string,
+  userType: PropTypes.string,
+  age: PropTypes.number,
+  gender: PropTypes.string,
+  secretWords: PropTypes.string,
+  secretPictureName: PropTypes.string,
+  secretPicturePath: PropTypes.string,
+  sectionId: PropTypes.number,
+});
+
+const GENDERS = {
+  'm': i18n.genderMale(),
+  'f': i18n.genderFemale()
+};
+
+/** @enum {number} */
+export const COLUMNS = {
+  NAME: 0,
+  AGE: 1,
+  GENDER: 2,
+  PASSWORD: 3,
+  ACTIONS: 4,
+};
+
+export const styles = {
+  table: {
+    width: 970,
```

Comment:
Should we be using styleConstants['content-width']?

Reply:
Yes, but my next PR is a refactor of a bunch of table styles, sorting, and components, so I'll fix it there. We have a few react tables now, and a lot of duplicate code.